src/orders/components/OrderSummaryCard/OrderSummaryCard.tsx

```diff
@@ -1,15 +1,14 @@
 // @ts-strict-ignore
```

Comment:
Suggestion: A bit unrelated, but looking at this component I think we can remove this